### DISCUSSION TITLE
Modal for "New send" gets closed without notice by clicking outside of it

### DISCUSSION THREAD

**Discussion by codiflow** [dani-garcia/vaultwarden#4148] [Q&A]
### Subject of the issue
If you create a new send and click outside of the modal the modal just closes no matter if you already interacted with it. So if you already uploaded files or entered text there and clicked outside by accident of the modal everything is gone and you have to start over again.

### Deployment environment
* vaultwarden version: 2023.10.0

* Install method: Docker image

* Clients used: Firefox Browser

* Reverse proxy and version: not relevant

* MySQL/MariaDB or PostgreSQL version: not relevant

* Other relevant details:

### Steps to reproduce
- Create a new send by clicking on "New send"
(- Enter some details)
- Click outside the modal

### Expected behaviour
After clicking outside (if you already entered data)
- either the modal only closes **after** asking you if you really want to close it
- or closing the modal by clicking outside is disabled completely
- or the values inside the fields are kept after closing and you can resume your work by clicking on "New send" again

### Actual behaviour
The modal closes without any notice or warning that data will be lost. Reopening the modal starts with all fields empty

### Troubleshooting data
-

**Comment by BlackDex** [comment]
This is a web-vault client specific issue. We do not maintain/develop the web-vault (on some small patches aside sometimes).
But this behavior is more general across all dialogs if I'm correct.

**Comment by codiflow** [comment]
Good to know, thanks for clarifying that. So this probably will never be solved this way. Thanks for letting me and others know :-)
